src/components/documentComponents/documentLinkSearch/ContentInnerModal.js

```diff
@@ -0,0 +1,51 @@
+import CommandList from "./CommandList.js";
+import DocumentLinkList from "./DocumentLinkList.js";
+
+export default function ContentInnerModal({ $target, selectionStart, option }) {
+  const $contentInnerModal = document.createElement("div");
+  $contentInnerModal.className = "content-inner-modal";
+  $contentInnerModal.contentEditable = false;
+  console.log(selectionStart);
+  if (selectionStart < 28) {
+    $contentInnerModal.style.left = `${selectionStart === 1 || 6 ? 0 : selectionStart + 60}px`;
+  } else {
+    $contentInnerModal.style.right = `${selectionStart + 22}px`;
+  }
+
+  $target.appendChild($contentInnerModal);
+
+  this.close = () => {
+    $contentInnerModal.remove();
+  };
+
+  if (option === "command") {
+    return new CommandList({
+      $parent: $target,
+      $target: $contentInnerModal,
+      onClose: () => {
+        const $textNode = $target.childNodes[0];
+        if ($textNode && $textNode.nodeType === Node.TEXT_NODE) {
+          const $text = $textNode.nodeValue;
+          const extractedText = $text.slice(-6);
+          const newText = $text.slice(0, -extractedText.length);
+          $textNode.nodeValue = newText;
+        }
+
+        this.close();
+      },
+    });
+  } else if (option === "link") {
+    return new DocumentLinkList({
+      $parent: $target,
+      $target: $contentInnerModal,
+      initialState: {
+        documentLinks: [],
+      },
+      onClose: () => {
+        this.close();
+      },
```

Comment:
원래 바로 함수를 할당하는데 this 바인딩 때문에 이렇게 됐군요ㅠ 선호 차이겠지만 아래처럼 하는건 어떤가요? this는 항상 성가신 문제네요
```suggestion
onClose: this.close.bind(this),
```

Reply:
bind를 쓰는 방법도 있군요. 그런데 이 방식하고 제안주신 방식의 차이가 무엇인지 알고싶은데...레퍼런스라도 공유해주실 수 있으실까요?

Reply:
두 코드 모두 this의 컨텍스트를 올바르게 유지하기 위해 사용되는 패턴입니다. `onClose: () => { this.close(); };` 여기서는 화살표 함수를 사용하여 새로운 함수를 생성합니다. 화살표 함수는 자체적으로 자신의 this를 가지고 있지 않습니다. 그렇기 때문에 자신의 상위 스코프의 this 값을 참조합니다.
`onClose: this.close.bind(this);` 여기서는 bind 메서드를 사용하여 this.close 함수의 this 값을 현재의 this 값으로 고정시킵니다. 
저는 this binding을 위해 한 뎁스 더 들어가는 것을 별로 선호하지 않아서 `bind`를 자주 사용하지만 둘 다 결국 this 컨텍스트를 유지하기 위해 사용하는 것은 동일합니다.